elixir/apps/domain/lib/domain/auth/provider/query.ex

```diff
@@ -14,10 +14,6 @@ defmodule Domain.Auth.Provider.Query do
     where(queryable, [provider: provider], is_nil(provider.disabled_at))
   end
 
-  def not_exceeded_attempts(queryable \\ not_deleted()) do
```

Comment:
Just curious why this isn't needed anymore?

Reply:
It wasn't used

Reply:
Ah, ok, sounds good.  Do we have any automated alerting on failed syncs?

Reply:
Yes, we have alerts for everything that logs an `error`. So we will alert after 3 failed attempts.

Reply:
Awesome